.gitignore

```diff
@@ -0,0 +1,2 @@
+example/index.html
+.DS_Store
```

Comment:
We should probably leave system files out of the `.gitignore`. People typically set up a global `.gitignore` to handle this:

https://help.github.com/articles/ignoring-files/